### PR TITLE
Corrects strings in json error responses

### PR DESCRIPTION
**Pull request by jrforrest**
Addresses #1037

### CODE REVIEW

**Comment by jrforrest** [comment]
Looks like this is breaking on some platforms.  Investigating...

File: lib/grape/error_formatter/json.rb [hunk 1]
```diff
@@ -3,14 +3,23 @@ module ErrorFormatter
...
             result = result.merge(backtrace: backtrace)
           end
           MultiJson.dump(result)
         end
+
+        private
+
+        def wrap_message(message)
+          if message.is_a?(Exceptions::ValidationErrors)
+            return message
```
Comment (dblock) [inline lib/grape/error_formatter/json.rb]: Remove `return`, that's implied.  Expand the condition to be consistent:

```ruby
if message.is_a?(...)

elsif message.respond_to?(:to_h)

else

end
```

Or better

```
case
when ...
else
...

I am not in love with the check for `to_h`, I think you want to check `is_a?(Hash)` - or are you seeing a Mash or something? In which case I think it should still be checked explicitly.

Comment (jrforrest) [inline lib/grape/error_formatter/json.rb]: Yeah you're right that conditional was sloppy.  Corrected.  Will squash if this looks better.

**Comment by dblock** [comment]
It looks better, squash it.

**Comment by dblock** [comment]
Merged via https://github.com/ruby-grape/grape/commit/844b6ace4f2298db8892bd79954d1ac64d9d59da.

**Pull request closed without merging**